cache-config/t3cutil/getdatacfg.go

```diff
@@ -794,10 +794,12 @@ func GetConfigData(toClient *toreq.TOClient, disableProxy bool, cacheHostName st
 		return true
 	})
 
-	err := error(nil)
-	toData.ServerParams, err = atscfg.GetServerParameters(toData.Server, combineParams(toData.ServerProfilesParams))
-	if err != nil {
-		errs = append(errs, err)
+	if len(errs) == 0 && toData.Server != nil {
```

Comment:
`errs` needs to be empty in order to append to it?

Reply:
No, but if there's an error, `toData.Server` may be nil, which is the panic here. 

The idiomatically-correct thing to do is to stop processing any data if there are any errors, i.e. `len(errs) == 0`.
Theoretically, if `len(errs) == 0` then `toData.Server` should never be nil here; the `toData.Server != nil` is just an additional safety, before dereferencing it.